### PR TITLE
BNGP-4307: Create a Service down/Shutter static HTML page 

### PR DESCRIPTION
**Pull request by usman-j-shaikh**

### CODE REVIEW

File: packages/webapp/src/routes/__tests__/shutter-page.spec.js
Comment (paulsimonandrews) [inline packages/webapp/src/routes/__tests__/shutter-page.spec.js:1]: Remove this file, it's not needed. The shutter page isn't served by the web app.

File: packages/webapp/src/routes/shutter-page.js
Comment (paulsimonandrews) [inline packages/webapp/src/routes/shutter-page.js:1]: Remove this file, it's not needed. The shutter page isn't served by the web app.

File: packages/webapp/src/utils/constants.js
Comment (paulsimonandrews) [inline packages/webapp/src/utils/constants.js:1]: Remove these edits from constants, they are not needed. The shutter page isn't served by the web app.

File: packages/webapp/src/views/shutter-page.html
Comment (paulsimonandrews) [inline packages/webapp/src/views/shutter-page.html:1]: The shutter page isn't a nunjucks templated page but a standalone static HTML page

**Review by paulsimonandrews — CHANGES_REQUESTED** [review]
The shutter page should be a static HTML page, not a nunjucks templated page. 

It's a standalone page that isn't served by the web app but loaded into the cloud firewall when the web app is unreachable.

Please follow the instructions on the JIRA ticket

**Review by KiranVarma02 — COMMENTED** [review]
should file name be down.html like example for some reason as example in ticket, or is it not necessary?

**Review by KiranVarma02 — APPROVED** [review]

**Review by KiranVarma02 — APPROVED** [review]

**Comment by LeenaKu** [comment]
Tested and Passed

**Review by KiranVarma02 — APPROVED** [review]

**Review by AjinkyaJawalkar — APPROVED** [review]

**Review by paulsimonandrews — APPROVED** [review]

**Pull request merged**